src/main/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -662,7 +662,7 @@ private ReadEndsForMarkDuplicates buildReadEnds(final SAMFileHeader header, fina
      * Goes through the accumulated ReadEndsForMarkDuplicates objects and determines which of them are
      * to be marked as duplicates.
      */
-    private void generateDuplicateIndexes(final boolean useBarcodes, final boolean indexOpticalDuplicates) {
+        private void generateDuplicateIndexes(final boolean useBarcodes, final boolean indexOpticalDuplicates) {
```

Comment:
revert indent change
```suggestion
    private void generateDuplicateIndexes(final boolean useBarcodes, final boolean indexOpticalDuplicates) {
```